data/com/mojang/blaze3d/font/TrueTypeGlyphProvider.mapping

```diff
@@ -1,7 +1,13 @@
 CLASS com/mojang/blaze3d/font/TrueTypeGlyphProvider
+	METHOD <init> (Ljava/nio/ByteBuffer;Lorg/lwjgl/stb/STBTTFontinfo;FFFFLjava/lang/String;)V
+		ARG 1 fontMemory
+		ARG 2 font
+		ARG 4 oversample
 	METHOD getGlyph (I)Lcom/mojang/blaze3d/font/TrueTypeGlyphProvider$Glyph;
 		ARG 1 character
 	CLASS Glyph
+		METHOD <init> (Lcom/mojang/blaze3d/font/TrueTypeGlyphProvider;IIIIFFI)V
+			ARG 8 index
```

Comment:
I believe you have not counted the implicit first parameter for the instance itself. There's two implicit (not seen in source) parameters at the beginning: the instance of itself (`Glyph`) which is not shown in descriptors, and the instance of the owning outer class (`TrueTypeGlyphProvider`), which is shown in the descriptor. So that becomes parameter 0 (as parameter indexes start from 0) and parameter 1, respectively.